pkg/datastore/test/tuples.go

```diff
@@ -1595,10 +1598,11 @@ func ConcurrentWriteSerializationTest(t *testing.T, tester DatastoreTester) {
 				close(waitToStart)
 			})
 			<-waitToFinish
-
 			return err
 		})
-		require.NoError(err)
+		if err != nil {
+			panic(err)
```

Comment:
I know it's a test, but why panic when it's already in an errgroup?

Reply:
Before I changed it, if an error was raised, the test runner would fail but not say why (because the require executes in a goroutine). The `panic` forces the error to be shown in full